Sources/NetworkRequestHelper.swift

```diff
@@ -209,7 +209,7 @@ class NetworkRequestHelper {
     }
 }
 
-extension NetworkRequest {
+public extension NetworkRequest {
```

Comment:
Should we update the access levels for the methods as well? Applies to the other classes in this PR